centreon/www/front_src/src/Dashboards/Dashboard/Layout/Panel/Panel.tsx

```diff
@@ -46,15 +53,24 @@ const Panel = ({ id }: Props): JSX.Element => {
     ) : (
       <FederatedComponent
         isFederatedWidget
+        canEdit={canEditField}
         globalRefreshInterval={refreshInterval}
         id={id}
+        isEditing={isEditing}
         panelData={panelOptionsAndData?.data}
         panelOptions={panelOptionsAndData?.options}
         path={panelConfigurations.path}
+        saveDashboard={saveDashboard}
```

Comment:
I'm aware that this prop hasn't been introduced in this PR but I don't think that any prop should be specific to the dashboard component

Reply:
It's to improve to improve UX on the map widget. It avoid to save the widget then save the dashboard